web-console/src/entry.ts

```diff
@@ -73,7 +73,7 @@ if (consoleConfig.baseURL) {
   UrlBaser.baseUrl = consoleConfig.baseURL;
 }
 if (consoleConfig.customHeaderName && consoleConfig.customHeaderValue) {
-  apiConfig.headers.common[consoleConfig.customHeaderName] = consoleConfig.customHeaderValue;
+  apiConfig.headers[consoleConfig.customHeaderName] = consoleConfig.customHeaderValue;
```

Comment:
https://github.com/apache/druid/pull/10597/files#diff-5563ae2b9ba7d6595882c5bae584d1b9db3db64c0257d8d661ccd98e6a6029f0L77 It looks like `.common` was a copy over from this PR

I don't understand the web console enough to comment on this change. Can you explain what broke/ what tests we need to prevent this from breaking in the future

Reply:
I think the default axios config has `.common` key on headers (meaning headers that should be used for all requests). https://github.com/axios/axios#global-axios-defaults but the new defaults did not. I think we can set all these `consoleConfig` configs as part of the e2e tests to make sure nothing breaks as a result.

Reply:
cool 👍  - thanks for the explanation!